todo_app/data/view_model.py

```diff
@@ -0,0 +1,33 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def done_items(self):
+        items = []
+        for item in self._items:
+            status = item.status
+            if status == 'Done':
+                items.append(item)
+    
+        return items
```

Comment:
Not necessarily recommended, but I do think it's a good opportunity to compare this (arguably more readable approach) with a more concise one-liner list comprehension; this translates directly to something like:
```python
return [item for item in self._items if item.status == 'Done']
```